collectors/plugins.d/gperf-config.txt

```diff
@@ -1,64 +1,112 @@
+%{
+
+#define PLUGINSD_KEYWORD_ID_FLUSH                  97
+#define PLUGINSD_KEYWORD_ID_DISABLE                98
+#define PLUGINSD_KEYWORD_ID_EXIT                   99
+#define PLUGINSD_KEYWORD_ID_HOST                   71
+#define PLUGINSD_KEYWORD_ID_HOST_DEFINE            72
+#define PLUGINSD_KEYWORD_ID_HOST_DEFINE_END        73
+#define PLUGINSD_KEYWORD_ID_HOST_LABEL             74
+
+#define PLUGINSD_KEYWORD_ID_BEGIN                  12
+#define PLUGINSD_KEYWORD_ID_CHART                  32
+#define PLUGINSD_KEYWORD_ID_CLABEL                 34
+#define PLUGINSD_KEYWORD_ID_CLABEL_COMMIT          35
+#define PLUGINSD_KEYWORD_ID_DIMENSION              31
+#define PLUGINSD_KEYWORD_ID_END                    13
+#define PLUGINSD_KEYWORD_ID_FUNCTION               41
+#define PLUGINSD_KEYWORD_ID_FUNCTION_RESULT_BEGIN  42
+#define PLUGINSD_KEYWORD_ID_FUNCTION_PROGRESS      43
+#define PLUGINSD_KEYWORD_ID_LABEL                  51
+#define PLUGINSD_KEYWORD_ID_OVERWRITE              52
+#define PLUGINSD_KEYWORD_ID_SET                    11
+#define PLUGINSD_KEYWORD_ID_VARIABLE               53
+#define PLUGINSD_KEYWORD_ID_CONFIG                 100
+
+#define PLUGINSD_KEYWORD_ID_CLAIMED_ID             61
+#define PLUGINSD_KEYWORD_ID_BEGIN2                 2
+#define PLUGINSD_KEYWORD_ID_SET2                   1
+#define PLUGINSD_KEYWORD_ID_END2                   3
+
+#define PLUGINSD_KEYWORD_ID_CHART_DEFINITION_END   33
```

Comment:
Does `gperf` has an option to order these values?

Reply:
No, we have to do it manually. It's my intention to do this in a separate PR, because this one is already complex as it is.